### ISSUE TITLE
What is this about?

### ISSUE THREAD

**Issue by abebeos** [henryjfry/repository.thenewdiamond#24]
Me new to kodi.

Could you describe in one sentence + one paragraph what this repository is about and why I should use it?

**Comment by henryjfry** [comment]
This repo is a replacement for the original extendedinfo addon for kodi, thats its in one sentence. But if you dont know kodi that wont mean a lot.

https://kodi.tv/addons/nexus/script.extendedinfo/

Extendedinfo is used by a number of skins to provide special information screens.
(see the TL/DR at the bottom)

However in addition to fixing the addon when it stopped working after the change to python3 in kodi 19 I have added a lot of additional functionailty.

So the way I use the addon, I have it set to load straight into my Trakt Recently watched tv list (which is a setting you can enable in the settings to autoload any of the addons paths) on startup.
This loads in a nice UI which shows posters and information.  This can be done with many addons but the kodi screens are often pretty slow (if you arent running it on a pc youll notice pretty quickly) and even with cached data the screens are slow to progress, reload etc.

So this loads its own XML screens which dont stop and have persistent python running to do things (which basically any other kodi screen wont unless you are running a big fancy skin).
Therefore the screens never really close and the cached information which loads is the absolute minimum and therefore browsing/reloading is a lot faster after the initial information lookup.

The extendedinfo UI therfore provides a quick and nice looking interface which alows you to browse TMDB your Trakt lists/trending/popular/collection/watched list/in progress/next episode plus a couple of pages I cooked up myself based on your trakt data.
Also IMDB lists and a youtube browser.

So in one place you can browse a tonne of nice looking screens for basically any lists you might have setup and do searches for people/shows/movies and discover from TMDB with a bunch of filters and sorting options.
From there you can then play via tmdbhelper to any addon you have installed.
And it will play, trakt scrobble and reopen the addon on finish.

It also provides tmdbhelper based trakt collection library integration (a lot of skins work better with library files).
But this is not really necessarry if you use trakt and RD through my addon.

It also offers RD RSS integration and a number of other "hidden" features, which I use and im not sure anyone else does.
I would document them but i get next to no help testing anything so i generally dont feel like writing up docs if no one in the community is willing to provide any assistance.
Also I'd have to make sure the stuff actually worked properly if I advertised and im quite happy with the halfbaked stuff ive bolted on.
But I can point you in the direction of the one file where you can basically find everything usable i've implemented and you can play about with it yourself.

tl/dr
This is at its core an information provider/browser for kodi which provides a nice and very usable interface to browse and discover tv shows/movies and can be used to play them back with tmdbhelper.

![screenshot-01](https://github.com/henryjfry/repository.thenewdiamond/assets/44173765/040ffbe4-0840-454c-af1f-d70ba391a39a)

**Comment by henryjfry** [comment]
If you want me to give you a little overview on what you would need to do to start using this yourself and watching stuff I can do.
Its not a 1 app install like it would be on android.

There are a couple of steps to get stuff up and running but its pretty straightforward after that.

**Comment by abebeos** [comment]
> tl/dr This is at its core an information provider/browser for kodi which provides a nice and very usable interface to browse and discover tv shows/movies and can be used to play them back with tmdbhelper.

**thenewdiamond** is a video manager for kodi which provides an easy to use, fast and beautiful interface to discover, organize and view your favorite tv shows and movies.

> This repo is a replacement for the original extendedinfo addon for kodi, thats its in one sentence. But if you dont know kodi that wont mean a lot.
> 
> https://kodi.tv/addons/nexus/script.extendedinfo/

=> extendedinfo based (ui-less utility script for kodi)
=> repo contains much more, "kodi video heaven"


### Extendedinfo 
> Extendedinfo is used by a number of skins to provide special information screens. (see the TL/DR at the bottom)
> 
> However in addition to fixing the addon when it stopped working after the change to python3 in kodi 19 I have added a lot of additional functionailty.
> 
> So the way I use the addon, I have it set to load straight into my Trakt Recently watched tv list (which is a setting you can enable in the settings to autoload any of the addons paths) on startup. This loads in a nice UI which shows posters and information. This can be done with many addons but the kodi screens are often pretty slow (if you arent running it on a pc youll notice pretty quickly) and even with cached data the screens are slow to progress, reload etc.

#### background scripts

> So this loads its own XML screens which dont stop and have persistent python running to do things (which basically any other kodi screen wont unless you are running a big fancy skin). Therefore the screens never really close and the cached information which loads is the absolute minimum and therefore browsing/reloading is a lot faster after the initial information lookup.

#### ui

> The extendedinfo UI therfore provides a quick and nice looking interface which alows you to browse TMDB your Trakt lists/trending/popular/collection/watched list/in progress/next episode plus a couple of pages I cooked up myself based on your trakt data. Also IMDB lists and a youtube browser.
> 

#### tmdb based
> So in one place you can browse a tonne of nice looking screens for basically any lists you might have setup and do searches for people/shows/movies and discover from TMDB with a bunch of filters and sorting options. From there you can then play via tmdbhelper to any addon you have installed. And it will play, trakt scrobble and reopen the addon on finish.
> 
> It also provides tmdbhelper based trakt collection library integration (a lot of skins work better with library files). But this is not really necessarry if you use trakt and RD through my addon.

#### RD RSS integration

> It also offers RD RSS integration and a number of other "hidden" features, which I use and im not sure anyone else does.

#### status: Works, hobby, fuzzy

I would document them but i get next to no help testing anything so i generally dont feel like writing up docs if no one in the community is willing to provide any assistance. Also I'd have to make sure the stuff actually worked properly if I advertised and im quite happy with the halfbaked stuff ive bolted on. But I can point you in the direction of the one file where you can basically find everything usable i've implemented and you can play about with it yourself.
> 

> 
> ![screenshot-01](https://private-user-images.githubusercontent.com/44173765/297500521-040ffbe4-0840-454c-af1f-d70ba391a39a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU1NDk2MjMsIm5iZiI6MTcwNTU0OTMyMywicGF0aCI6Ii80NDE3Mzc2NS8yOTc1MDA1MjEtMDQwZmZiZTQtMDg0MC00NTRjLWFmMWYtZDcwYmEzOTFhMzlhLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE4VDAzNDIwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjMGI2YjdlMzdjYWQ0MjAyZjMzNTEzNTU1Y2ZhMWUxOThlZmEzZDBiNDdlNzExZjczNGI5MjVkN2M0NjZhNDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.92UEjj6vCHHOyCGlcYc78cR2A5ncnqmaskAerLc_KqI)

**Comment by abebeos** [comment]
> If you want me to give you a little overview on what you would need to do to start using this yourself and watching stuff I can do.

Only if you write it down in a setup doc.

What are alternatives to your repo, with similar functionality?

**Comment by henryjfry** [comment]
Like what's all this about??
Did you ask chat gpt to summarise this, why?

> # repository.thenewdiamond
> 
> > This is at its core an information provider/browser for kodi which provides a nice and very usable interface to browse and discover tv shows/movies and can be used to play them back with tmdbhelper.
> 
> **thenewdiamond** is a video manager for kodi which provides an easy to use, fast and beautiful interface to discover, organize and view your favorite tv shows and movies.
> 
> > This repo is a replacement for the original extendedinfo addon for kodi, thats its in one sentence. But if you dont know kodi that wont mean a lot.
> > https://kodi.tv/addons/nexus/script.extendedinfo/
> 
> => extendedinfo based (ui-less utility script for kodi) => repo contains much more, "kodi video heaven"
> ### Extendedinfo
> 
> > Extendedinfo is used by a number of skins to provide special information screens. (see the TL/DR at the bottom)
> > However in addition to fixing the addon when it stopped working after the change to python3 in kodi 19 I have added a lot of additional functionailty.
> > So the way I use the addon, I have it set to load straight into my Trakt Recently watched tv list (which is a setting you can enable in the settings to autoload any of the addons paths) on startup. This loads in a nice UI which shows posters and information. This can be done with many addons but the kodi screens are often pretty slow (if you arent running it on a pc youll notice pretty quickly) and even with cached data the screens are slow to progress, reload etc.
> 
> #### background scripts
> 
> > So this loads its own XML screens which dont stop and have persistent python running to do things (which basically any other kodi screen wont unless you are running a big fancy skin). Therefore the screens never really close and the cached information which loads is the absolute minimum and therefore browsing/reloading is a lot faster after the initial information lookup.
> 
> #### ui
> 
> > The extendedinfo UI therfore provides a quick and nice looking interface which alows you to browse TMDB your Trakt lists/trending/popular/collection/watched list/in progress/next episode plus a couple of pages I cooked up myself based on your trakt data. Also IMDB lists and a youtube browser.
> 
> #### tmdb based
> 
> > So in one place you can browse a tonne of nice looking screens for basically any lists you might have setup and do searches for people/shows/movies and discover from TMDB with a bunch of filters and sorting options. From there you can then play via tmdbhelper to any addon you have installed. And it will play, trakt scrobble and reopen the addon on finish.
> > It also provides tmdbhelper based trakt collection library integration (a lot of skins work better with library files). But this is not really necessarry if you use trakt and RD through my addon.
> 
> #### RD RSS integration
> 
> > It also offers RD RSS integration and a number of other "hidden" features, which I use and im not sure anyone else does.
> 
> #### status: Works, hobby, fuzzy
> 
> I would document them but i get next to no help testing anything so i generally dont feel like writing up docs if no one in the community is willing to provide any assistance. Also I'd have to make sure the stuff actually worked properly if I advertised and im quite happy with the halfbaked stuff ive bolted on. But I can point you in the direction of the one file where you can basically find everything usable i've implemented and you can play about with it yourself.
> 
> > 
> 
> > ![screenshot-01](https://private-user-images.githubusercontent.com/44173765/297500521-040ffbe4-0840-454c-af1f-d70ba391a39a.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU1NDk2MjMsIm5iZiI6MTcwNTU0OTMyMywicGF0aCI6Ii80NDE3Mzc2NS8yOTc1MDA1MjEtMDQwZmZiZTQtMDg0MC00NTRjLWFmMWYtZDcwYmEzOTFhMzlhLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE4VDAzNDIwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNjMGI2YjdlMzdjYWQ0MjAyZjMzNTEzNTU1Y2ZhMWUxOThlZmEzZDBiNDdlNzExZjczNGI5MjVkN2M0NjZhNDMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.92UEjj6vCHHOyCGlcYc78cR2A5ncnqmaskAerLc_KqI)

**Comment by abebeos** [comment]
> , I suggest you browse the stickies here:

I asked you for a suggestion.

What? I edited your reply.

Anyways, I give it up. This repositoriy is totally confusing, and it looks like you prefer to keep it that way.

All good.

**Comment by henryjfry** [comment]
Yeah I thought you were asking me Kodi related questions and you were going to be starting to use it and potentially wanted to try my addon too 
Not that you were prodding to document my repo.

Which really isn't a thing.
I might document the addon better. But it's a Kodi repository in a git hub repository

They are both called the same thing but you don't "use my repo". You install the stuff from my repo and then you use the addon from my repo.

The questions you were asking made it clear you haven't actually used Kodi.
There is literally loads of information on how to get started. Until you had a look at that there's no point you trying to decide why you should use my addon and I can't really explain why until you've tried some of the others.

**Comment by abebeos** [comment]
You can delete the issue.
